Akilahinternees/Andelachallenge
# Modify UI

> #### What does this PR do?
> Modifying UI
>
> #### Description of Task to be completed
> I change home page of the project, change colors on other pages
>
> #### How should this be manually tested?
>You can view the code run them to see if they meet expectations
> #### Any background context you want to provide?
>N/A
> #### What are the relevant pivotal tracker stories?
>ft-modify-interface-168781427
> #### Screenshots (if appropriate)
>N/A
